src/types/Project.ts

```diff
@@ -5,4 +5,5 @@ export type Project = {
   categories: Array<Category>;
   name: string;
   images: Array<Image>;
+  tileSize: number;
```

Comment:
`tileSize` should be moved to the root reducer because we use `state.project` for serialization (i.e. “Save project”). I recommend adding a new `settingsSlice` for these type of properties and actions.

Reply:
Yeah, wasn't sure about where to put the new variable and whether we want to persist after save/load.